bug_report_page.php

```diff
@@ -545,8 +542,8 @@
 <?php
 	}
 
-	# Display tags fields
-	if( $t_show_tags ) { ?>
+	# Display tags fields if user is allowed to attach tags
```

Comment:
I would remove the added comment.  Again it is not just about being allowed, but about the fields configured to show in the report page.